frontend/uddhar-frontend/src/authentication/services/auth.jsx

```
@@ -0,0 +1,117 @@
+    return {
+      status: false,
+      message: error?.response?.data?.message || "An error occurred while logging in.",
+    };
+  }
+};
+
+export const setHeader = (token) => {
+  if (token) {
+    axios.defaults.headers.common["Authorization"] = `Bearer ${token}`;
+  } else {
+    delete axios.defaults.headers.common["Authorization"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)